packages/mpv.rb

```diff
@@ -3,64 +3,93 @@
 class Mpv < Package
   description 'Video player based on MPlayer/mplayer2'
   homepage 'https://mpv.io/'
-  version '0.32.0'
+  version '0.33.0'
   license 'LGPL-2.1+, GPL-2+, BSD, ISC and GPL-3+'
   compatibility 'all'
-  source_url 'https://github.com/mpv-player/mpv/archive/v0.32.0.tar.gz'
-  source_sha256 '9163f64832226d22e24bbc4874ebd6ac02372cd717bef15c28a0aa858c5fe592'
-
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-x86_64.tar.xz',
+  source_url "https://github.com/mpv-player/mpv/archive/v#{version}.tar.gz"
+  source_sha256 'f1b9baf5dc2eeaf376597c28a6281facf6ed98ff3d567e3955c95bf2459520b4'
 
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: 'ae9f812ccb3239ca85d25a6e919e1ded6e459867ba00d2027ea832782a38f9f1',
-     armv7l: 'ae9f812ccb3239ca85d25a6e919e1ded6e459867ba00d2027ea832782a38f9f1',
-       i686: '41f96ce28132e2d60978c3e7450594b58b03fda67edbb8fa2e56ce6442d3d1ee',
-     x86_64: 'e9623b3fdd06ab5bb762a15a3d7ef9b7875a10d4f0a5baaed5f83b9250b9fbe4',
-
+  binary_sha256({
+    aarch64: '834b7660f6c44bc5a52236828f6451e03da7f80ce7dcaef84aa5bf539209a0fb',
+     armv7l: '834b7660f6c44bc5a52236828f6451e03da7f80ce7dcaef84aa5bf539209a0fb',
+       i686: 'e0bdd8907d0f3673397adcc6d042e878510a6657e347fcc4c30db9bdf40af33d',
+     x86_64: 'ae78493b16e21e2c9b8ac6ff794a4a89305f9f3686392961b4277bcd5175d109'
   })
 
-  depends_on 'ld_default' => :build
+  depends_on 'alsa_lib'
   depends_on 'docutils'
   depends_on 'ffmpeg'
-  depends_on 'gdbm'
+  depends_on 'jack'
   depends_on 'lcms'
   depends_on 'libarchive'
+  depends_on 'libass'
+  depends_on 'libbluray'
   depends_on 'libcaca'
   depends_on 'libcdio'
+  depends_on 'libcdio_paranoia'
+  depends_on 'libdrm'
+  depends_on 'libdvdnav'
+  depends_on 'libdvdread'
+  depends_on 'libjpeg'
+  depends_on 'libjpeg_turbo'
+  depends_on 'libva'
+  depends_on 'libvdpau'
+  depends_on 'libx11'
+  depends_on 'libxext'
+  depends_on 'libxinerama'
+  depends_on 'libxkbcommon'
+  depends_on 'libxrandr'
+  depends_on 'libxss'
   depends_on 'libxv'
   depends_on 'luajit'
+  depends_on 'mesa'
+  depends_on 'mujs'
+  depends_on 'pipewire'
   depends_on 'pulseaudio'
-  depends_on 'uchardet'
-  depends_on 'xdg_base'
+  depends_on 'rubberband'
+  depends_on 'shaderc'
   depends_on 'sommelier'
```

Comment:
Can we list sommelier last?

Reply:
I've been alphabetizing these with `sort -u` so I don't miss any/ get dupes. Is there a reason to put it higher or lower?

Reply:
The reason we need to list sommelier last is so users won't miss the all-important postinstall message.  If packages are installed afterwards, the message will scroll past the visible screen area.